tasks/package.js

```diff
@@ -96,9 +168,12 @@ module.exports = function(grunt) {
       }
     });
 
-    grunt.config.set('clean.packages', [destPath]);
-
-    tasks.push('clean:packages');
+    if (useRsync) {
+      tasks.push('mkdir:package');
+      tasks.push('shell:rsync');
+    } else {
+      tasks.push('copy:source');
+    }
     tasks.push('copy:package');
```

Comment:
That seems fine. This is a stepping stone to #255, otherwise I'd suggest we drop projFiles and rename the config for clarity.